### ISSUE TITLE
How can I tell the dropdown that it should render error styling?

### ISSUE THREAD

**Issue by JPStrydom** [fateh999/react-native-paper-dropdown#93]
Most of the react-native-paper input components take in an `error` prop which gives the input an error styling. Look at react-native-paper's [`TextInput` component](https://callstack.github.io/react-native-paper/docs/components/TextInput/) or react-native-paper-dates' [`DatePickerInput` component](https://web-ridge.github.io/react-native-paper-dates/docs/date-picker/input-date-picker).

Is there a way to achieve the same with react-native-paper-dropdown?

**Comment by aubryll** [comment]
@JPStrydom You can do it like this:
 ```
<DropDown
      ...
      inputProps={{
        error: true|false,
      }}/>
```

**Comment by fateh999** [comment]
Should be fixed in latest release v2.0.3, did a full refactor of library so docs are in WIP, meanwhile you can check the example folder for demo code.
